### ISSUE TITLE
JLCPCB caan not produce the chamfered edges

### ISSUE THREAD

**Issue by MarcSN311** [TobleMiner/HPE-FlexibleLOM-adapter#1]
I just tried to order the PCBs from JLCPCB. They wont produce the chamfered edges because the minimum required board size for the chaamfer is 50mm. 
Maybe you should add that to the Readme to make the ordering process smoother.

**Comment by DanielG** [comment]
I also ordered some boards from JLC just now, but they seem to have accepted them for production without a hitch. Though you are right, their [docs](https://jlcpcb.com//quote/pcbOrderFaq/Gold%20Fingers) say "3. A board with a size of smaller than 5x5cm will be too small to be beveled.". Weird.

**Comment by TobleMiner** [comment]
Interesting. I did not run into that issue.
I could panelize the boards, connecting two at the long non-beveled side. IIRC they can bevel opposing edges.

**Comment by TobleMiner** [comment]
@MarcSN311 
I've panelized two of the boards. Feel free to try uploading this file to JLC:
[fab_panel.zip](https://github.com/TobleMiner/HPE-FlexibleLOM-adapter/files/5323663/fab_panel.zip)

**Comment by MarcSN311** [comment]
@DanielG they accepted my production files as well but then reached out to me to tell me they could not do it. 
@TobleMiner Thanks, i'll give it a try.

**Comment by TobleMiner** [comment]
@MarcSN311 Did JLC approve the files yet? I'd love to update the repo with a panelized version.

**Comment by footlongC** [comment]
> @MarcSN311 Did JLC approve the files yet? I'd love to update the repo with a panelized version.

I was able to get an order in and approved yesterday. Shows in production now.

**Comment by MarcSN311** [comment]
Sorry I'm a bit slow. I will inform you as soon as my production files are confirmed.

**Comment by TobleMiner** [comment]
No worries. I'm not in a hurry. Just want to make sure you are keeping us all up to date :smile:

**Comment by psc24** [comment]
I've had fab_panel.zip rejected because it had too many files; I assume this is the embedded fab_panel directory. Having removed that from the zip it seems to have been accepted.

**Comment by neobenedict** [comment]
I was able to order revA.zip quantity 10 pieces without issue. It might be you weren't ordering a large enough quantity.

**Comment by psc24** [comment]
Nope, I ordered 10 off as well. Based on experience so far, I think it's just as likely that there's little consistency in the checks JLC perform!

Hi Sir/ Madam, there are too many files in your zip file and we don’t know which file we should use, so could you please delete useless files in your Zip file?You can click ‘Replace file’ button to re-upload the file in your JLCPCB account page .Thanks.

**Comment by TobleMiner** [comment]
Oops, the subdirectory is not supposed to be in that zip file. I've updated the zip file above again (actually final this time, I promise!) so JLC won't be confused by it anymore.

**Comment by MarcSN311** [comment]
Just received my order!
I had the same problem with the duplicate files, but they did not complain about the chamfered edges anymore. 
Sadly they did not produce the chamfered edges anyways -.-

**Comment by neobenedict** [comment]
Same thing here, no chamfers, is it an issue?

**Comment by MarcSN311** [comment]
I think the chafers are just nicer to your PCIe slots. I am just annoyed by thefact that they exchanged a bunch of messages with me about the chafer and in the end they did not produce it anyways.

**Comment by laf0rge** [comment]
Indeed, the production run JLC did for me also silently disregarded the chamfered edges. I think with careful insertion into the slots it shouldn't be that much of a problem.  Aside from that, there's always the option to file the edges by hand, but FR4 dust is also not particularly healthy, to say the least...

**Comment by kontakt** [comment]
I ordered 5x of the latest and got received 4, one did not survive fab. May be related to difficulty in chamfering.

**Comment by MarcSN311** [comment]
@kontakt were the remaining 4 PCBs chamfered?

**Comment by kontakt** [comment]
Yes, the four I received were chamfered.

**Comment by TobleMiner** [comment]
@neobenedict It is not a particularly big issue. However it makes insertion easier and increases the lifetime of the PCIe slot.

**Comment by jbaznik** [comment]
Is this order HPE FlexibleLOM to PCIe adapter from https://cart.jlcpcb.com by your notes for ordering ok.

[cart.jlcpcb.com-PCB Prototype amp PCB Fabrication Manufacturer - order - HPE-FlexibleLOM-adapter.pdf](https://github.com/TobleMiner/HPE-FlexibleLOM-adapter/files/5757713/cart.jlcpcb.com-PCB.Prototype.amp.PCB.Fabrication.Manufacturer.-.order.-.HPE-FlexibleLOM-adapter.pdf)

**Comment by TobleMiner** [comment]
Other than that I'd suggest selecting "Specify a location"at the bottom of the page this order looks correctly configured to me.

**Comment by 0mnicy** [comment]
How much did your order of 10 cost you? I'm trying to use oshpark

**Comment by TobleMiner** [comment]
Depends on whether you want 10 boards or 10 adapters, since there are 2 adapters per board. With edge chamfer and ENIG the prices at JLC are:

5 boards: $20
10 boards: $25

Most of the price is a one-time fee for surface finish and edge chamfer.

**Comment by 0mnicy** [comment]
> 10 boards or 10 adapters

The difference between the two are what? The gerber files you upload on the site? Boards are fab_panel.zip and adapters are revA.zip?

I tried uploading the fab_panel.zip and got a quote for $29.20 (qty 10) with the settings are shown on the pdf from jbaznik. That sound right or should i upload revA.zip and do the same settings from jbaznik?

**Comment by TobleMiner** [comment]
Yes, the panel contains two adapter boards each. However, getting the panel made costs the same as ordering just single boards.

$29 sounds about right? Does that price include shipping? My previous numbers were probably e bit off since I guesstimated them from  Euros.

**Comment by 0mnicy** [comment]
So the grand total is $44 with fab_panel.zip. The boards were $29 and shipping with Fedex was $15. I added gold fingers to try and reduce some oxidation over time but that was probably not needed (i didn't see a price increase with that option).  TobleMiner do you have a way to contact you outside this github issue?

**Comment by TobleMiner** [comment]
@the1cyrus Sure, feel free to write me either DMs on Twitter (https://twitter.com/Toble_Miner) or contact me by E-Mail: tobleminer(at)gmail(dot)com

**Comment by 0mnicy** [comment]
Just fyi for everyone.. JLCPCB charged me an extra $25 because I used the paneled boards (fab_panel.zip). 
"Hi Sir/Madam,the special price is only for the single board and not for the panel board,so you may need to pay for an extra cost for it."

**Comment by TobleMiner** [comment]
Unfortunately review is very inconsistent at JLC. This kind of stuff seems to depend heavily on who exactly reviews your gerber files. Not that I'd recommend this, but sometimes just cancelling the order and resubmitting it might work wonders.

**Comment by CNC-at-Home** [comment]
Hi together, I've a question please, I ordered some of the pcbs from sysmocom.de and then I attached it with the LAN-Card (it's a HPE FlexFabric 10Gb 2-port 533FLR-T)  into the motherboard, but no reaction from Windows 10. Maybe is there a need to set jumpers or should it works out of the box, only soldering the pins. I've a second card for my Linux Server, but I will wait to build in until my question is answered. Thank you

Greetings
Olli
